src/init.cpp

```diff
@@ -1225,6 +1225,16 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                     LogPrintf("Prune: pruned datadir may not have more than %d blocks; -checkblocks=%d may fail\n",
                         MIN_BLOCKS_TO_KEEP, GetArg("-checkblocks", 288));
                 }
+
+                {
+                    LOCK(cs_main);
+                    CBlockIndex* tip = chainActive.Tip();
+                    if (tip && tip->nTime > GetAdjustedTime() + 2 * 60 * 60) {
+                        strLoadError = _("The block database contains a block which appears to be from the future. This may be due to your computer's date and time being set incorrectly. Only rebuild the block database if you are sure that your computer's date and time are correct");
```

Comment:
@bensussman To answer your question: no, there is no such rule. See https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md 

@casey Using multiple "line" "line2" line3" does not affect translation IIRC if you specify only one `_`, eg:
```c++
strLoadError = _("The block database contains a block which appears to be from the future. "
    "This may be due to your computer's date and time being set incorrectly. "
    "Only rebuild the block database if you are sure that your computer's date and time are correct");
```